Sources/MockoloFramework/Models/TypeAliasModel.swift

```diff
@@ -69,6 +84,12 @@ final class TypeAliasModel: Model {
     }
     
     func render(with identifier: String, typeKeys: [String: String]? = nil) -> String? {
+        if processed {
+            if let modelDescription = modelDescription {
+                return modelDescription
+            }
```

Comment:
so if it's processed but there no modelDescription, does that mean something is wrong ? Should there be an `else` here ?